site/content/documentation/tutorials/getting-started.md

```diff
@@ -138,18 +140,17 @@ Not very pretty perhaps, but at least you should be able to recognize the RDF st
 
 ### Example 02: using the ModelBuilder
 
-The previous code example shows that you need to do a bit of preparation before actually adding anything to your model: defining common namespaces, creating a ValueFactory, creating IRIs, etc. As a convenience, RDF4J provides a {{< javadoc "ModelBuilder" "model/util/ModelBuilder.html" >}} that simplifies things.
+The previous code example shows that you need to do a bit of preparation before actually adding anything to your model: defining common namespaces, creating IRIs, etc. As a convenience, RDF4J provides a {{< javadoc "ModelBuilder" "model/util/ModelBuilder.html" >}} that simplifies things.
 
 {{< example "Example 02" "model/Example02BuildModel.java" >}} shows how we can create the exact same model using a ModelBuilder:
 
 ```java {linenos=inline}
 ModelBuilder builder = new ModelBuilder();
-Model model = builder
-                  .setNamespace("ex", "http://example.org/")
-		  .subject("ex:Picasso")
-		       .add(RDF.TYPE, "ex:Artist")
-		       .add(FOAF.FIRST_NAME, "Pablo")
-		  .build();
+Model model = builder.setNamespace("ex", "http://example.org/")
+      .subject("ex:Picasso")
+      .add(RDF.TYPE, "ex:Artist")
+      .add(FOAF.FIRST_NAME, "Pablo")
+      .build();
```

Comment:
I think the original formatting was better.

Reply:
I deliberately changed it because the formatting as we had it was rather artificial. It might have been slightly easier to read with custom indentation but most code editors will reformat to use conformant indentation, as shown here.